docs/console.html

```diff
@@ -0,0 +1,804 @@
+    </p>
+        <dl>
+            <dt>Hide network messages</dt>
+            <dd>Instructs the console to not show logs from network issues. For example, 404 and 50x errors will not be displayed.</dd>
+            
+            <dt>Log XMLHttpRequests</dt>
+            <dd>Determines if each XMLHttpRequest is logged to the Console panel.</dd>
+            
+            <dt>Preserve log upon navigation</dt>
+            <dd>Determines if the console history for the current page is preserved when you navigate to another page.</dd>
+            
+            <dt>Show timestamps</dt>
+            <dd>Displays the timestamp when the message was logged. This will disable <a href="#message-stacking">message stacking</a>.</dd>
+        </dl>
+
+
+    <h3 id="message-stacking">Message Stacking</h3>
+    <p>
+        The console will stack consecutive messages that contain the same output.
+        This helps keep the information provided to you as concise as possible.
+    </p>
+    
+    <figure>
+        <table>
+            <thead>
+                <tr>
+                    <th>Timestamps disabled (default)</th>
+                    <th>Timestamps enabled</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><img src="console-files/timestamps-disabled.png" alt="Timestamps disabled"/></td>
+                    <td><img src="console-files/timestamps-enabled.png" alt="Timestamps enabled"/></td>
+                </tr>
+            </tbody>
+        </table>
+        <figcaption>
+            Examples of the two stacking states.
+        </figcaption>
+    </figure>
+    
+    <figure>
+        <figcaption>
+            Sample code for testing console stack modes.
+        </figcaption>
+        <pre class="prettyprint"><code class="language-javascript">
+msgs = ['hello', 'world', 'there'];
+for (i = 0; i < 20; i++) console.log(msgs[Math.floor((i/3)%3)]) 
+</code></pre>
+    </figure>
+</div>
+
+<div class="collapsible">
+        <h2 id="using-the-console-api">Using the Console API</h2>
+        <p>
+            The Console API is collection of methods provided by the global `console` object defined by DevTools.
+            One of the API's main purposes is to log information to the console while your application is running.
+            You can also group output visually in the console to reduce visual clutter.
+        </p>
+
+        <section>
+            <h3 id="writing-to-the-console">Writing to the console</h3>
+            <p>
+                The <a href="console-api#consolelogobject-object">console.log()</a> method takes one or more expressions as parameters and writes their current values to the console.
+            </p>
+
+            <figure id="simple-console-write">
+                <figcaption>A simple write to the console.</figcaption>
+                <figure>
+                    <pre class="prettyprint"><code class="language-javascript">
```

Comment:
we end up with an extra newline
![image](https://cloud.githubusercontent.com/assets/39191/6384212/66d82ecc-bd0e-11e4-9a94-2a01f71260bd.png)

would love to nuke that.